functions/block-patterns/elemente/person-3.html

```diff
@@ -109,10 +109,6 @@
 <!-- /wp:sunflower/meta-data --></div></div>
 <!-- /wp:group --></div></div>
 <!-- /wp:media-text -->
-
-<!-- wp:paragraph -->
-<p></p>
-<!-- /wp:paragraph --></div>
```

Comment:
I don't think it's correct to remove the closing `</div>` here. The formatting of this file is rather confusing. Don't ask me, why.

Reply:
Doesn't work.

1. copy your code into the code editor of a new page
2. switch to the editor
3. you see two code block repair buttons, click it
4. copy the code from the code editor
5. you will find the closing `</div>` now behind the coment `<!-- /wp:media-text -->`

The code editor is causing this little funny formatting.

![2024-07-10_20-59](https://github.com/verdigado/sunflower/assets/3354500/e009557c-a76f-4f5a-a7ef-ee0bc81c39cc)

Reply:
You are right. It should work now.